modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/backend/MBIntegrationUiBaseTest.java

```diff
@@ -129,6 +129,28 @@ protected void restartServerWithDifferentAdminRoleName() throws Exception {
                 "user-mgt.xml"), true, true);
     }
 
+
+    /**
+     *
+     * Restart server with altered maximumRedeliveryAttempts configuration in broker.xml
+     *
+     * @throws Exception
+     */
+    protected void restartServerWithAlteredMaximumRediliveryAttempts() throws Exception {
```

Comment:
Can we avoid throwing generic exception ? is this due to the exception thrown by automation framework ?